### ISSUE TITLE
Steam API Key Issue

### ISSUE THREAD

**Issue by masonotaku** [ztizzlegaming/CSGOWinBig#178]
Hello! Awesome source code, by the way.

I'm not storing my api key in a .txt file, as i'm just testing. I'll show you what I've done: 

[https://imgur.com/y0gITwn](url)

This method is not working. I've directly pasted my api key into the function. What have i done wrong?

Thanks in advance.

**Comment by andrewda** [comment]
That function will just set $apikey to your ApiKey, I'm not sure what you expect it to do.

**Comment by masonotaku** [comment]
Yes, I know that. It's still not linking to my api key when i click "login with steam". :/

**Comment by mvegter197** [comment]
http://pastebin.com/8ZCNq7f8 did you try all these steps?

**Comment by masonotaku** [comment]
Ah, thank you! That's perfect.

**Comment by masonotaku** [comment]
Sadly, it's still saying csgowinbig when i click 'Sign in through Steam'.

**Comment by mvegter197** [comment]
Could you upload the content of your steamauth/settings.php (remove apikey) to pastebin.com?

**Comment by masonotaku** [comment]
Ok. I'll do so. Thanks for your help. I'm new to PHP.

**Comment by masonotaku** [comment]
http://pastebin.com/cL5u8asL

**Comment by mvegter197** [comment]
Okay, that is fine, did you edit the index.php? at lines: 54 & 120? 
There are two csgowinbig.com which you have to replace in each line.

**Comment by masonotaku** [comment]
Where is the index.php file? Inside steam auth? I get `<?php header('Location: /'); ?>` when i click on that. 

Sorry for the trouble.

**Comment by mvegter197** [comment]
My fault, I mean index.html not .php

**Comment by masonotaku** [comment]
On lines 54 & 120 I have: 

1. ` <a href="https://steamcommunity.com/openid/login?openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0&openid.mode=checkid_setup&openid.return_to=http%3A%2F%2Fwww.csgowinbig.com%2Fphp%2Fsteam-login-handle.php&openid.realm=http%3A%2F%2Fwww.csgowinbig.com&openid.ns.sreg=http%3A%2F%2Fopenid.net%2Fextensions%2Fsreg%2F1.1&openid.claimed_id=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select&openid.identity=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select">`

2. `a href="https://steamcommunity.com/openid/login?openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0&openid.mode=checkid_setup&openid.return_to=http%3A%2F%2Fwww.csgowinbig.com%2Fphp%2Fsteam-login-handle.php&openid.realm=http%3A%2F%2Fwww.csgowinbig.com&openid.ns.sreg=http%3A%2F%2Fopenid.net%2Fextensions%2Fsreg%2F1.1&openid.claimed_id=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select&openid.identity=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select" class="btn btn-lg btn-success" id="login-main" style="display: none; margin-bottom: 10px;">`

What do I change? Thanks! I apologise for my lack of knowledge.

**Comment by mvegter197** [comment]
You could have found the mistake if you read those lines:
"openid.return_to=http%3A%2F%2Fwww.csgowinbig.com%2Fphp%2Fsteam-login-handle.php"

you need to replace www.csgowinbig.com with www.masonfirman.com/eggybetting

**Comment by masonotaku** [comment]
Sorry. Thank you for your help! Do I need to change this on every page?

**Comment by mvegter197** [comment]
Yes

**Comment by masonotaku** [comment]
Ok, cheers!
